monai/apps/detection/transforms/dictionary.py

```diff
@@ -95,6 +99,50 @@
 DEFAULT_POST_FIX = PostFix.meta()
 
 
+class StandardizeEmptyBoxd(MapTransform, InvertibleTransform):
```

Comment:
I'm not familiar with the use cases, is there a case that the user don't want to use this transform... maybe this transform should be enabled by default within some existing transforms, for example in  `MaskToBoxD`?

Reply:
Good point. How about running it by default inside the detector?

Reply:
sure, perhaps call the util in the detector https://github.com/Can-Zhao/MONAI/blob/c13895d021b0d5aa4caccc85f742a7dac4b1cc89/monai/data/box_utils.py#L519

Reply:
I let the detector automatically correct the empty box shape and label data type, while print a warning message to let the users know what has been done. Shall we remove the array and dict transforms?

Reply:
looks good to me!